packages/hub/src/communication/index.ts

```diff
@@ -0,0 +1,118 @@
+import {SignedState} from '@statechannels/nitro-protocol';
```

Comment:
Do we need to import _all_ of these? The entire file?

Reply:
I think so. The way I constructed this file (on my branch, which Alex copied across) was to add the minimum and then keep adding until that compiled. We should be able to reduce this a lot as we firm up the wire protocol, but I think this is what's needed to decouple in the short-term.